player/command.c

```diff
@@ -3353,23 +3355,20 @@ static int mp_property_packet_bitrate(void *ctx, struct m_property *prop,
     if (r[type] < 0)
         return M_PROPERTY_UNAVAILABLE;
 
-    // r[type] is in bytes/second -> bits
-    double rate = r[type] * 8;
-
-    // Same story, but used kilobits for some reason.
-    if (old)
-        return m_property_int64_ro(action, arg, llrint(rate / 1000.0));
+    // r[type] is in bytes/second
+    double rate = r[type];
 
     if (action == M_PROPERTY_PRINT) {
-        rate /= 1000;
-        if (rate < 1000) {
-            *(char **)arg = talloc_asprintf(NULL, "%.f kbps", rate);
+        rate /= 1024;
+        if (rate < 1024) {
+            *(char **)arg = talloc_asprintf(NULL, "%.f KiBps", rate);
         } else {
-            *(char **)arg = talloc_asprintf(NULL, "%.3f Mbps", rate / 1000.0);
+            *(char **)arg = talloc_asprintf(NULL, "%.3f MiBps", rate / 1024);
```

Comment:
IMHO `KiB/s` looks less cursed than `KiBps`.

But it also more common to use bits per second. So maybe we should stick to that, to avoid angry users coming at us. And kilobits per seconds, not kibibits.

I know I suggested using `utils.format_bytes_humanized`, but it seems to do more harm than good. Unless we actually change there.

So to summary. I would use `bits`, `kb/s`, `Mb/s` for bitrates. That would be compatible with common values that other software show.